views/spoty/artist-search-result.hbs

```diff
@@ -0,0 +1,20 @@
+
+<div class="container ">
+    <h1 class="text-center my-5 text-white">Resultados para {{artistArr.name}}</h1>
+        <div class="row ">
+            {{#each artistArr as |artist|}}
+            <div class="col-4">
+                <div class="column card m-3 bg-secondary">
+                  <img src={{artistImg}} alt="" style="width:100%; height:150px; background-color:white" class="mx-auto my-4 rounded bg-dark">
```

Comment:
En la vista de artistas lo haría así:
```
                  {{#if artist.img}}
                    <img src="{{artist.img}}" alt="{{artist.name}}" style="width:100%; height:150px; background-color:white" class="mx-auto my-4 rounded bg-dark">
                  {{else}}
                    <img src="https://picsum.photos/200" alt="{{artist.name}}" style="width:100%; height:150px; background-color:white" class="mx-auto my-4 rounded bg-dark">
                  {{/if}}
```

La imagen del else en vez de usar un lorem quizás quedaría mejor con una imagen de ejemplo para todas con el logo de la app o algo genérico gris que ponga not image.

Por otro lado cambiaría también el uso de style con width y height fijos a img-fluid 

```
<img src="{{artist.img}}" alt="{{artist.name}}"  class="img-fluid mx-auto my-4 rounded bg-dark">
```